### PR TITLE
Feature: lesson10

### PR DESCRIPTION
**Pull request by haru-programming**
# [Issue No.10](https://github.com/kenmori/handsonFrontend/blob/0087a459ae7e84af59ead2d97d9c6b5153481e69/work/markup/1.md#10)
- 問題9(async/awaitを使ったローディング実装)に追加で try-catch-finaliy を書いてください
## Concerns and areas to focus on
- I'm particularly worried about the way the catch error is written.

### CODE REVIEW

File: lesson10/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        fetchListData();
+    } catch {
+        const error = 'エラー：データが取得できません';
+        console.error(error);
+    } finally {
+        removeLoading();
+    }
+};
+
+handlingException();
```
Comment (kenmori) [inline lesson10/index.js:40]: これは意味がない気がしました
fetchListDataは確かにリクエストしていますが、その結果何をやるわけでもなさそうです

`const values = await fetchListData();`
ここに対してのtry catchをしないといけない気がしました

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
[the remark this answers is not in the document]
Comment (haru-programming) [reply .vscode/settings.json]: Thank you for your review!
I added .gitignore file.
From now on, I'll be careful to only upload files to git that I need!

-> a0478c4

File: lesson10/index.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+async function addList() {
+    const fragment = document.createDocumentFragment();
+
+    const values = await fetchListData();
```
Comment (seeeen044) [inline lesson10/index.js:45]: How about receiving this with try and making this function just a function that generates a list tag?

File: lesson10/index.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    removeLoading();
```
Comment (seeeen044) [inline lesson10/index.js:46]: It doesn't have to be here because it is processed in finally.

File: lesson10/index.js (hunk 1 above)
Comment (haru-programming) [reply lesson10/index.js]: Thank you for the review!
After reading your explanation,
 I found out that the error message was not caught in the handlingException function.
 I tried to fix it in my own way.

9af180e

File: lesson10/index.js (hunk 3 above)
Comment (haru-programming) [reply lesson10/index.js]: Thank you for being so specific!!
I changed the way I assembled the addList function from the try-catch statement in my own way.

9af180e

File: lesson10/index.js (hunk 4 above)
Comment (haru-programming) [reply lesson10/index.js]: I was able to notice that the execution of removeLoading was duplicated.
Thank you! 
Sorry about the commits being together. 
I have removed the relevant part.

9af180e

**Review by seeeen044 — APPROVED** [review]

**Comment by haru-programming** [comment]
@kenmori 
Based on the reviews I received on Discord,
 I made some corrections.
 It was a great learning experience.
Thank you very much for actually writing the code.

bcc429d

**Pull request merged**